__TESTS__/unit/actions/Overlay.test.ts

```diff
@@ -239,6 +240,30 @@ describe('Tests for overlay actions', () => {
     expect(asset.toURL()).toContain("l_text:arial_15:!/fl_layer_apply");
   });
 
+  it("textFit with alias", () => {
+    const asset = createNewImage('sample');
+    const text = 'hello hello hello hello';
+    const textSource = Source.text(text, 'arial_15').textFit(TextFit.size(400, 500));
+    asset.overlay(Overlay.source(textSource));
+    expect(asset.toURL()).toContain("c_fit,w_400,h_500");
```

Comment:
please add l_sample to the expected result so we make sure it's not in a different component